tests/tensor/test_basic.py

```diff
@@ -922,6 +922,22 @@ def check_u(m, k=0):
             assert np.allclose(result_indx, result_from)
             assert result.dtype == np.dtype(dtype)
 
+        def check_l_batch(m, k=0, tensor_type=matrix):
+            m_symb = tensor_type(dtype=m.dtype)
+            k_symb = iscalar()
+            f = function([m_symb, k_symb], tril(m_symb, k_symb))
+            result = f(m, k)
+            assert np.allclose(result, np.tril(m, k))
+            assert result.dtype == np.dtype(dtype)
+
+        def check_u_batch(m, k=0, tensor_type=matrix):
```

Comment:
I would not add defaults, they make the tests less readable if arguments are not specified explicitly. 

In this case they are not even used, so you could actually have bugs/invalid values in the defaults and not know about it.

Reply:
In order to pass `tensor3` and `tensor4` to the same function, I introduced this default value.
Is it okay if we only check for `tensor3` and not `tensor4`? If so, I'll remove the default and make this function only check for `tensor3`.

Reply:
yes that should be fine